### ISSUE TITLE
Failed to get python threadid for main thread when profiling a Dockerized process

### ISSUE THREAD

**Issue by vainu-lauri** [benfred/py-spy#168]
When profiling a python process from host that is running in a Docker container I'm getting lots of these errors:

```
[2019-09-23T12:04:34.576469893Z ERROR py_spy::python_spy] failed to get python threadid for main thread!
[2019-09-23T12:04:34.607087353Z ERROR py_spy::python_spy] failed to get python threadid for main thread!
```
Occasionally (like one in 1000 lines) I'm able to see the top view of py-spy for a single sample

This used to work with previous version of py-spy and the issue started occurring with the new 0.2.0 version.

In the container I'm running `Python 2.7.15+` and the host is Debian 9.9 running kernel:
`4.9.0-9-amd64 #1 SMP Debian 4.9.168-1+deb9u5 (2019-08-11) x86_64 GNU/Linux`

**Comment by benfred** [comment]
So - the problem is that we're spamming the log with a non-fatal error message here, and thats getting in the way of the top view =(  As a short term fix if you suppress all log messagse with ```RUST_LOG=none py-spy top --pid 12345``` I think everything should still work as before.

I'll get a proper fix in the next release -

**Comment by benfred** [comment]
fixed in v0.2.1
